starjudge2/main.py

```diff
@@ -7,9 +7,9 @@
 
         
 POINTS = 50
-A = 25
-B = 50
-C = -abs(25)
+A = 20
+B = 15
+C = -abs(10)

```

Comment:
```suggestion
C = -10
```
I think we can simply type `-10` without calling the `abs` function.

Reply:
I wasn't sure because i haven't work that much on python but for sure is a good idea